Libraries/Components/ScrollResponder.js

```diff
@@ -405,6 +405,31 @@ var ScrollResponderMixin = {
   },
 
   /**
+   * Scrolls to the end of the ScrollView, either immediately or with a smooth
+   * animation.
+   *
+   * Example:
+   *
+   * `scrollResponderScrollToEnd({animated: true})`
+   */
+  scrollResponderScrollToEnd: function(
+    options?: { animated?: boolean },
+  ) {
+    if (Platform.OS !== 'ios') {
+      console.warn(
+        'scrollResponderScrollToEnd is not supported on this platform'
+      );
+      return;
+    }
+    const animated = (options && options.animated === true) ? true : false;
```

Comment:
`options && options.animated === true` evaluates to `undefined` when no options object is passed. This causes a crash in native because native expects a boolean. Will add an explanatory comment.